examples.js

```diff
@@ -0,0 +1,199 @@
+(function () {
+
+  function addVlExample(path, id, options) {
+    d3.json(path, function(error, vlSpec) {
+      if (error) {
+        return console.warn(error);
+      }
+      var stackedbarEmbed = {
+        mode: "vega-lite",
+        spec: vlSpec
+      };
+      vg.embed(id, stackedbarEmbed, function(error, result) {
+        vlTooltip(result.view, vlSpec, options);
+      });
+    });
+  }
+
+  function addVgExample(path, id, options) {
+    d3.json(path, function(error, vgSpec) {
+      if (error) {
+        return console.warn(error);
+      }
+      vg.parse.spec(vgSpec, function(error, chart) {
+        var view = chart({el:id}).update();
+        vgTooltip(view, options);
+      });
+    })
+  }
+
+  /* Vega-Lite Examples */
+  // Scatter Plot
+  var scatterOpts = {
+    showFields: [
+      {
+        field: "Name"
+      },
+      {
+        field: "Horsepower",
+        type: "number"
+      },
+      {
+        field: "Miles_per_Gallon",
+        fieldTitle: "Miles per Gallon",
+        type: "number"
+      }
+    ]
+  };
+  addVlExample("exampleSpecs/scatter.json", "#vis-scatter", scatterOpts);
+
+  // Trellis Barley
+  var trellisBarOpts = {
+    showFields: [
+      {
+        field: "mean_yield",
```

Comment:
I think user should not have to know that we have `mean_yield` under the hood.   (We never expose underscore prefix elsewhere in Vega-Lite)